src/Machine.Specifications/Explorers/AssemblyExplorer.cs

```diff
@@ -48,7 +48,7 @@ public IEnumerable<ISupplementSpecificationResults> FindSpecificationSupplements
         public IEnumerable<IAssemblyContext> FindAssemblyContextsIn(Assembly assembly)
         {
             return assembly.GetExportedTypes()
-              .Where(x => x.GetInterfaces().Contains(typeof(IAssemblyContext)))
+              .Where(x => x.GetTypeInfo().IsClass && x.GetTypeInfo().IsAbstract == false && x.GetInterfaces().Contains(typeof(IAssemblyContext)))
```

Comment:
Simplify to just `!x.GetTypeInfo().IsAbstract`

Reply:
done - I prefer == false. Easier to see